pkg/client/options.go

```diff
@@ -23,6 +23,11 @@ import (
 	"k8s.io/apimachinery/pkg/selection"
 )
 
+const (
+	// DefaultPageLimit represents the default limit used when specifyinh the 'Page' ListOption.
+	DefaultPageLimit = 100
```

Comment:
This probably needs a better name since it's not the default for List() too.

Reply:
@vincepri @cben WDYT?